src/qthotkey/plugin.h

```diff
@@ -0,0 +1,53 @@
+#ifndef _PLUGIN_H_INCLUDED_
+#define _PLUGIN_H_INCLUDED_
+
+#include <QtCore/QList>
+
+namespace GlobalHotkeys {
+
+enum EVENT {
```

Comment:
Use `enum Event` and camelcase enums like `Play`, `PrevTrack`, etc.  Then you can refer to your events as `Event::Play`, etc.

Reply:
I'll update it.